e2e/react_native_windows_vnext/__tests__/bundling.test.ts

```diff
@@ -0,0 +1,82 @@
+import path from 'path';
+import fs from 'fs';
+import { installDeps, cleanup } from '../../../integration_tests/utils/common';
+import { runHaulSync } from '../../../integration_tests/utils/runHaul';
+import {
+  validateBaseBundle,
+  validateHostBundle,
+  validateAppBundle,
+} from '../../utils/bundles';
+
+const PROJECT_FIXTURE = path.join(
+  __dirname,
+  '../../../fixtures/react_native_windows_vnext'
+);
+
+describe('react_native_windows_vnext', () => {
```

Comment:
these tests for windows current and vnext are quite repetitive, how about extract helpers and pass paths etc accordingly?